### ISSUE TITLE
git error in new repository

### ISSUE THREAD

**Issue by edublancas** [ploomber/ploomber#1067]
```sh
ploomber examples -n templates/ml-basic -o ml
cd ml
git init
ploomber status
```

Output:

```
Loading pipeline...
fatal: bad revision 'HEAD'
name      Last run          Outdated?      Product            Doc (short)       Location
--------  ----------------  -------------  -----------------  ----------------  -----------------
get       Has not been run  Source code    File('output/get.  Get data          /Users/eduardo/De
                                           parquet')                            sktop/test/ml/tas
                                                                                ks.py:6
features  Has not been run  Source code &  File('output/feat  Generate new      /Users/eduardo/De
                            Upstream       ures.parquet')     features from     sktop/test/ml/tas
                                                              existing columns  ks.py:20
join      Has not been run  Source code &  File('output/join  Join raw data     /Users/eduardo/De
                            Upstream       .parquet')         with generated    sktop/test/ml/tas
                                                              features          ks.py:29
fit       Has not been run  Source code &  MetaProduct({'mod  Train a model     /Users/eduardo/De
                            Upstream       el': File('output                    sktop/test/ml/fit
                                           /model.pickle'),                     .py
                                           'nb': File('outpu
                                           t/nb.html')})
```

The `fatal: bad revision 'HEAD'` is due to an error when calling git, looks like isn't causing any trouble (besides the error message), we should check.

**Comment by edublancas** [comment]
the error might also appear in existing repos so we should investigate

**Comment by edublancas** [comment]
fixed in 066511a1e875c052fc0f4bc01785c0c9e3e99a6c
